projects/1.19/assets/more-beautiful-plates/morebeautifulplates/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+{
+  "block.morebeautifulplates.acacia_log_plate": "金合欢原木压力版",
+  "block.morebeautifulplates.birch_log_plate": "白桦木原木压力板",
+  "block.morebeautifulplates.black_concrete_plate": "黑色混凝土压力板",
+  "block.morebeautifulplates.black_concrete_powder_plate": "黑色混凝土粉末压力板",
+  "block.morebeautifulplates.black_glazed_terracotta_plate": "黑色带釉陶瓦压力板",
+  "block.morebeautifulplates.black_shulker_box_plate": "黑色潜影盒压力板",
+  "block.morebeautifulplates.black_stained_glass_plate": "黑色染色玻璃压力板",
+  "block.morebeautifulplates.black_terracotta_plate": "黑色陶瓦压力板",
+  "block.morebeautifulplates.black_wool_plate": "黑色羊毛压力板",
+  "block.morebeautifulplates.block_of_quartz_plate": "石英压力板",
+  "block.morebeautifulplates.blue_concrete_plate": "蓝色混凝土压力板",
+  "block.morebeautifulplates.blue_concrete_powder_plate": "蓝色混凝土粉末压力板",
+  "block.morebeautifulplates.blue_glazed_terracotta_plate": "蓝色带釉陶瓦压力板",
+  "block.morebeautifulplates.blue_shulker_box_plate": "蓝色潜影盒压力板",
+  "block.morebeautifulplates.blue_stained_glass_plate": "蓝色染色玻璃压力板",
+  "block.morebeautifulplates.blue_terracotta_plate": "蓝色陶瓦压力板",
+  "block.morebeautifulplates.blue_wool_plate": "蓝色羊毛压力板",
+  "block.morebeautifulplates.bone_block_plate": "骨块压力板",
+  "block.morebeautifulplates.bookshelf_plate": "书架压力板",
+  "block.morebeautifulplates.bricks_plate": "红砖块压力板",
+  "block.morebeautifulplates.brown_concrete_plate": "棕色混凝土压力板",
+  "block.morebeautifulplates.brown_concrete_powder_plate": "棕色混凝土粉末压力板",
+  "block.morebeautifulplates.brown_glazed_terracotta_plate": "棕色带釉陶瓦压力板",
+  "block.morebeautifulplates.brown_mushroom_block_plate": "棕色蘑菇方块压力板",
+  "block.morebeautifulplates.brown_shulker_box_plate": "棕色潜影盒压力板",
+  "block.morebeautifulplates.brown_stained_glass_plate": "棕色染色玻璃压力板",
+  "block.morebeautifulplates.brown_terracotta_plate": "棕色陶瓦压力板",
+  "block.morebeautifulplates.brown_wool_plate": "棕色羊毛压力板",
+  "block.morebeautifulplates.cactus_plate": "仙人掌压力板",
+  "block.morebeautifulplates.chiseled_quartz_block_plate": "錾制石英块压力板",
+  "block.morebeautifulplates.chiseled_red_sandstone_plate": "錾制红砂岩压力板",
+  "block.morebeautifulplates.chiseled_sandstone_plate": "錾制砂岩压力板",
+  "block.morebeautifulplates.chiseled_stone_bricks_plate": "錾制石砖压力板",
```

```suggestion
  "block.morebeautifulplates.chiseled_quartz_block_plate": "雕纹石英块压力板",
  "block.morebeautifulplates.chiseled_red_sandstone_plate": "雕纹红砂岩压力板",
  "block.morebeautifulplates.chiseled_sandstone_plate": "雕纹砂岩压力板",
  "block.morebeautifulplates.chiseled_stone_bricks_plate": "雕纹石砖压力板",
```
如果是以1193或1194翻译的话，此处需要更改